Remove History code

jorgelzpz opened the pull request:
Code wasn't working on browsers not supporting HTML5. After answering a question, those browsers try to request `http//host.tldundefined/post-id-N`

jorgelzpz committed:
Remove History code

ant30 commented:
It's already fixed in any newest askbot version? Could you tell us which version?

jorgelzpz committed:
Fix IE history problems in a more elegant way

jorgelzpz commented:
I found some references to this issue on commit ASKBOT/askbot-devel@610e7e34b8d9d01d7f6c0ffa02e2b20e5378e587, which is from August 29th. After studying the changes involved, I've moved this code to `javascript.html` and added a missing URL declaration.

This change has been tested under IE9 and I've made sure it works again.
